src/cmd/ksh93/tests/path.sh

```diff
@@ -879,7 +879,8 @@ got=$?
 	"(expected $exp, got $got)"
 
 # Tests for attempting to use a command name that's too long.
-long_cmd=$(awk -v ORS= 'BEGIN { for(i=0;i<500;i++) print "xxxxxxxxxx"; }')
+name_max=$(getconf NAME_MAX $PWD)
```

Comment:
Some old systems on which ksh works fine don't ship `getconf`. The built-in could be used instead but it may or may not be compiled in, so let's also add a reasonable fallback.
```sh
name_max=$(builtin getconf 2>/dev/null; getconf NAME_MAX . 2>/dev/null || echo 255)
```
(also using `.` instead of `"$PWD"` for brevity's sake; if `$PWD` is used, the expansion needs to be quoted.)